app/views/staff/activity_forms/purpose.html.haml

```diff
@@ -1,4 +1,7 @@
 = render layout: "wrapper" do |f|
   = f.govuk_fieldset legend: { text: t("form.legend.activity.purpose", level: t("page_content.activity.level.#{f.object.level}")), tag: :h1, size: "xl" } do
-    = f.govuk_text_field :title, label: { text: t("form.label.activity.title", level: t("page_content.activity.level.#{f.object.level}")).humanize }
+    = f.govuk_text_field :title,
+      label: { text: t("form.label.activity.title", level: t("page_content.activity.level.#{f.object.level}")).humanize },
+      hint_text: t("form.hint.activity.title", level: t("page_content.activity.level.#{f.object.level}"))
```

Comment:
Should the `hint_text` also be `humanize`d ?

Reply:
I think we are using humanize to simply capitalize the string, so here title needs capitalistion e.g. "Fund name" and the hint does not e.g. "A short title that explains the purpose of the fund".

I guess it should be `capitalize` in that case, but I am not sure it matters too much - depends on how much it offends you I guess?

Reply:
Fair enough, missed the difference between the hint and the label here 😟